admin/release.py

```diff
@@ -926,10 +916,6 @@ def create_release_branch_main(args, base_path, top_level):
         sys.stderr.write("%s: No (previous) pre-release exists for this "
                          "release.\n" % (base_path.basename(),))
         raise SystemExit(1)
-    except BaseBranchDoesNotExist:
```

Comment:
I think that this pattern for release tools causes problems. It might be suitable for user-facing tools.